agent/bench-scripts/tests/pbench-specjbb2005/test-56.txt

```diff
@@ -0,0 +1,183 @@
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/4-warehouses:4
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/4-warehouses:4/reference-result
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/4-warehouses:4/sample1
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/4-warehouses:4/sample1/SPECjbb.raw
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/4-warehouses:4/sample1/tools-default
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/5-warehouses:5
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/5-warehouses:5/reference-result
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/5-warehouses:5/sample1
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/5-warehouses:5/sample1/SPECjbb.raw
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/5-warehouses:5/sample1/tools-default
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/6-warehouses:6
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/6-warehouses:6/reference-result
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/6-warehouses:6/sample1
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/6-warehouses:6/sample1/SPECjbb.raw
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/6-warehouses:6/sample1/tools-default
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/7-warehouses:7
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/7-warehouses:7/reference-result
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/7-warehouses:7/sample1
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/7-warehouses:7/sample1/SPECjbb.raw
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/7-warehouses:7/sample1/tools-default
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/8-warehouses:8
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/8-warehouses:8/reference-result
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/8-warehouses:8/sample1
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/8-warehouses:8/sample1/SPECjbb.raw
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/8-warehouses:8/sample1/tools-default
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/SPECjbb.props
+/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-56_1900.01.01T00.00.00/SPECjbb.props.sed
```

Comment:
Should this file have been removed (or never created)?

Reply:
Nevermind:  I thought this was a backup of the original file produced before `sed` modified it (i.e., the result of the `-i` switch, which, apparently, we don't actually use...).  So, this is the actual `sed` script itself....  OK.